configure/osqp_configure.h.in

```diff
@@ -13,12 +13,17 @@ extern "C" {
 #cmakedefine IS_MAC
 #cmakedefine IS_WINDOWS
 
+/* ALGEBRA - Only one of the following is defined at compilation time */
+#cmakedefine ALGEBRA_DEFAULT
```

Comment:
We have been making any define that is visible to the external user prefixed with `OSQP_` to namespace our defines and ensure we won't conflict with anything in the future. We should do the same here, since this header is included in the main public header `osqp.h`. (we should fix the other defines in a separate PR).

Reply:
Opened a new issue #355 to keep track of this. Closing here.